src/__private_stories__/private-responsive-layout-scenarios-story.tsx

```diff
@@ -0,0 +1,230 @@
+                                                />
+                                            ))}
+                                        />
+                                    </ResponsiveLayout>
+                                </ResponsiveLayout>
+                            </ResponsiveLayout>
+                        </ResponsiveLayout>
+                    </ResponsiveLayout>
+                </ResponsiveLayout>
+
+                {/** Hero wrapped in multiple layouts */}
+                <ResponsiveLayout>
+                    <ResponsiveLayout>
+                        <ResponsiveLayout>
+                            <Hero
+                                background="default"
+                                media={<Image src={usingVrImg} aspectRatio="16:9" />}
+                                pretitle="Pretitle"
+                                title="Title"
+                                description="This is a long description with a long text to see how this works"
+                                desktopMediaPosition="right"
+                                button={
+                                    <ButtonPrimary small onPress={() => {}}>
+                                        Action
+                                    </ButtonPrimary>
+                                }
+                                buttonLink={<ButtonLink onPress={() => {}}>Link</ButtonLink>}
+                            />
+                        </ResponsiveLayout>
+                    </ResponsiveLayout>
+                </ResponsiveLayout>
+
+                {/** Slideshow with Hero wrapped in layout */}
+                <ResponsiveLayout>
+                    <Slideshow
+                        dataAttributes={{testid: 'slideshow'}}
+                        withBullets
+                        inverseBullets={false}
+                        items={Array.from({length: 3}, (_, idx) => (
+                            <Hero
+                                background="default"
+                                key={idx}
+                                media={<Image src={usingVrImg} aspectRatio="16:9" />}
+                                pretitle="Pretitle"
+                                title="Title"
+                                description="This is a long description with a long text to see how this works"
+                                desktopMediaPosition="right"
+                                button={
+                                    <ButtonPrimary small onPress={() => {}}>
+                                        Action
+                                    </ButtonPrimary>
+                                }
+                                buttonLink={<ButtonLink onPress={() => {}}>Link</ButtonLink>}
+                            />
+                        ))}
+                    />
+                </ResponsiveLayout>
+
+                {/** Multiple layouts */}
+                <ResponsiveLayout>
```

Comment:
Isn't this the same case you are testing with the Hero?

Reply:
Similar. In mobile, Hero uses a reset as wrapper, so it's not exactly the same